client/packages/common/src/utils/environment/EnvUtils.ts

```diff
@@ -18,6 +18,8 @@ const mapRoute = (route: string): RouteMapping => {
   switch (true) {
     case inRoute(AppRoute.Dashboard):
       return { title: 'dashboard', docs: '/getting_started/dashboard/' };
+    case inRoute(AppRoute.Reports):
+      return { title: 'reports', docs: 'getting_started/report/' };
```

Comment:
```suggestion
      return { title: 'reports', docs: '/getting_started/report/' };
```

All the others are specified with a leading `/` haven't checked if it works yet, but seems like it would be good to be consistent?